modules/pinger/pom.xml

```diff
@@ -47,7 +47,13 @@
     <dependency>
       <groupId>org.hawkular.inventory</groupId>
       <artifactId>inventory-api</artifactId>
-      <version>1.0.0-SNAPSHOT</version>
+      <version>0.0.1-SNAPSHOT</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>bus-api</artifactId>
```

Comment:
This needs to be changed to "inventory-bus-api" now that https://github.com/hawkular/hawkular-inventory/pull/49 is merged.